index.html

```diff
@@ -2,9 +2,169 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Rekupe agency</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+<header id="header" class="outer-content-container">
+    <div class="inner-content-container">
+        <nav class="header-content__navigation">
+            <ul>
+                <li><a href="#case-studies">Case Studies </a></li>
+                <li><a href="#services">Services</a></li>
+                <li><a href="#contact_us">Contact Us</a></li>
+            </ul>
+        </nav>
+
+        <div class="header-content__cta">
+            <h1 class="header-content__cta-items">Your website is an investment in the success of your business.</h1>
+            <p class="header-content__cta-items">We are here to help you succeed.</p>
+            <button type="button" class="header-content__cta-items">Let's work together!</button>
+        </div>
+
+    </div>
+    <div class="header__squiggle"></div>
+</header>
+
+<main >
+    <div class="outer-content-container">
+        <section class="main-content__section inner-content-container">
+            <article class="main-content-section1__article">
+                <p class="main-content__text1">Rekupe is a digital agency in Los Angeles.</p>
+                <p class="main-content__text1">We use strategic UX, SEO optimization and beautiful visual designs to
+                    help
+                    your business grow.</p>
+                <p class="main-content__text1">We see every project as a fresh opportunity pushing ourselves to create
+                    new
+                    digital experiences.</p>
+                <p class="main-content__text2">Have any questions? <a href="https://www.google.com">Click here</a> to
+                    contact us!
+                </p>
+            </article>
+        </section>
+    </div>
+    <div class="outer-content-container outer-content-container__grey-background">
+        <section class="main-content__section inner-content-container">
+            <div class="main-section2__container" id="case-studies">
+                <article class="main-section2__article_project">
+                    <div class="main-section2__project_textbox">
+                        <h2>Rolling Stone</h2>
+                        <div class="project_squiggle"></div>
+                        <p class="main-section2__project_description">You know Rolling Stone. It's the definitive voice
+                            in music, politics and culture, it's a global icon</p>
+                        <p class="main-section2__project_link">View project</p>
+                    </div>
+                    <span class="main-section2__project_image"><img
+                            src="/assets/work-image-01.jpeg"
```

Comment:
Dit pad werkt niet bij mij. Je zult óf "./assets" óf "assets" moeten gebruiken
 ```suggestion
                            src="./assets/work-image-01.jpeg"
```

```suggestion
                            src="assets/work-image-01.jpeg"
```